src/prefect/orion/models/work_queues.py

```diff
@@ -55,6 +55,13 @@ async def create_work_queue(
                     name=DEFAULT_AGENT_WORK_POOL_NAME, type="prefect-agent"
                 ),
             )
+            if work_queue.name == "default":
+                # If the desired work queue name is default, it was created when the
+                # work pool was created. We can just return it.
+                return await models.workers.read_work_queue(
```

Comment:
We're certain this is going to be the case and don't need any handling around this?

Reply:
Yes, a work queue named `default` is always created when a work pool is created.